### DISCUSSION TITLE
Use of operators in alpine ( x-show="edit && delete", for example)

### DISCUSSION THREAD

**Discussion by poutena** [alpinejs/alpine#2075] [1. Help] [answered]
Hello, everyone

I know this must be Alpine 101 for most of you, but I'm having trouble with using operators in Alpine and just couldn't find any information, neither on the documentation nor here or SO.

I'm trying to do something like this:
````HTML
<div x-data="{edit: false, delete: false}">
    <div id="show" x-show="!edit && !delete">…</div>
    <div id="edit" x-show="edit">…</div>
    <div id="delete" x-show="delete">…</div>
</div>
````
However, `x-show="!edit && !delete"` isn't working. I'm guessing this is not how Alpine works, but I just can't find information on how to do it. It might just be me not knowing how to ask the question, though.

Any help would be most appreciated.

**Comment by SimoTod** [comment, marked as answer]
The syntax is correct but delete is a reserved keyword in javascript (Alpine uses vanilla js under the hood). Try a different name
```html
<div x-data="{edit: false, del: false}">
    <div id="show" x-show="!edit && !del">1</div>
    <div id="edit" x-show="edit">2</div>
    <div id="delete" x-show="del">3</div>
</div>
```

**Reply by poutena** [reply]
Well, this is embarrassing… That's what having but a superficial knowledge of JS get you. Thank you!
